actions/service.go

```diff
@@ -23,6 +24,48 @@ type Task struct {
 	Hook   Hook
 }
 
+type TaskResult struct {
+	RunID      string
+	HookID     string
+	HookType   string
+	ActionName string
+	StartTime  time.Time
+	EndTime    time.Time
+	Passed     bool
+}
+
+type RunResult struct {
+	RunID     string
+	BranchID  string
+	OnRef     string
+	Action    *Action
```

Comment:
The `RunResults` captures a set of actions we execute - so we do not have any specific action information.
Think we will need the EventType - pre-commit, pre-merge and make it one of the key fields.

Reply:
Done.